samcli/commands/build/command.py

```diff
@@ -30,9 +30,10 @@
 \b
 Supported Runtimes
 ------------------
-1. Python2.7\n
-2. Python3.6\n
-3. Python3.7\n
+1. Python 2.7, 3.6, 3.7\n
+4. Nodejs 8.10, 6.10
+4. Ruby 2.5
+5. Java 8
```

Comment:
Do we need to call out which Dependency Managers we support?

Reply:
Hmm, yeah good idea. Will do